mmhuman3d/data/data_converters/agora.py

```diff
@@ -105,11 +120,50 @@ def convert_by_mode(self, dataset_path: str, out_path: str,
 
                     # obtain keypoints
                     keypoints2d = df.iloc[idx]['gt_joints_2d'][pidx]
+                    if self.res == '1280x720':
+                        keypoints2d *= (720 / 2160)
                     keypoints3d = df.iloc[idx]['gt_joints_3d'][pidx]
 
+                    gt_bodymodel_path = os.path.join(
+                        dataset_path,
+                        df.iloc[idx][f'gt_path_{self.fit}'][pidx])
+                    gt_bodymodel_path = gt_bodymodel_path.replace(
+                        '.obj', '.pkl')
+                    ann = pickle.load(open(gt_bodymodel_path, 'rb'))
+
+                    if self.fit == 'smplx':
+                        # obtain smplx data
+                        bodymodel['body_pose'].append(ann['body_pose'])
```

Comment:
let's all use `body_model` with the underscore.

Reply:
updated all instances in 829caa5